www/index.html

```diff
@@ -67,6 +67,10 @@
     <script src="lib/angularLocalStorage/dist/angularLocalStorage.min.js"></script>
     <script src="lib/angularjs-slider/dist/rzslider.min.js"></script>
     <script src="manual_lib/enketo/enketo-bundle.js"></script>
+
+    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+    <script src="https://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    <script type="text/javascript" src="cordova.js"></script>
```

Comment:
how do you know that these are the files needed for this to work? We already have jquery included in the index.html In general for hybrid apps, you don't want to use remote locations for code, since then the app won't work if the network is not available.